packages/angular/test/base/src/app/lazy/providers/providers.component.ts

```diff
@@ -28,7 +28,7 @@ export class ProvidersComponent {
     alertCtrl: AlertController,
     loadingCtrl: LoadingController,
     private menuCtrl: MenuController,
-    pickerCtrl: PickerController,
+    private pickerCtrl: PickerController,
```

Comment:
This diff shouldn't be needed. `pickerCtrl` is not being accessed outside of the constructor.